Meal creation

lutzcc1 opened the pull request:
Here's the meals controller with new-create-edit-update & destroy actions. Also the views that contain the forms to create and edit meals

jomilu93 commented on app/controllers/meals_controller.rb:
```diff
@@ -0,0 +1,44 @@
+  def create
+    @meal = Meal.new(meal_params)
+    if @meal.save
+      # redirect_to meal_path(@meal)
```
Hey Luis, here why did you comment out the redirect?

lutzcc1 replied:
Because that view still isn't ready. At the time I thought it would be good to comment it, now I realize it was just a dumb idea haha. Will push again with this un-commented

jomilu93 commented on app/controllers/meals_controller.rb:
```diff
@@ -0,0 +1,44 @@
+
+  def update
+    @meal.update(meal_params)
+    if @meal.save
```
Here...similar to how I did in the bookings controller, is the best way to do if meal.save or if meal.update?

lutzcc1 replied:
I was just testing if the rest of us are reading each others code... not haha. My bad, will push again

jomilu93 commented on app/controllers/meals_controller.rb:
```diff
@@ -0,0 +1,44 @@
+class MealsController < ApplicationController
+  before_action :set_meal, only: %i[edit update destroy]
```
I like this...it's cleaner and I think we should stick to this for all controllers.

lutzcc1 commented:
@jomilu93 

> Ummm I thought we need one file to create the form, which is the below file and then we render the form in the new.html.erb in the meals view folder, no? Why do we need the meal_form.html.erb file?

This was from an earlier commit, I corrected on the next one. I just realized that in that same commit I also added the corrections from your other comments... this GitHub thing still confuses me 🤔
